docs/databases/databases.html

```diff
@@ -102,29 +102,36 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/SwiftKueryMySQL">MySQL</a></li>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-SQLite">SQLite</a></li>
+          <li><a target="_blank" href="https://www.postgresql.org">PostgreSQL</a> using the <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL">Swift-Kuery-PostgreSQL</a> plugin.</li>
+          <li><a target="_blank" href="https://www.mysql.com">MySQL</a> using the <a target="_blank" href="https://github.com/IBM-Swift/SwiftKueryMySQL">SwiftKueryMySQL</a> plugin.</li>
+          <li><a target="_blank" href="https://www.sqlite.org">SQLite</a> using the <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-SQLite">Swift-Kuery-SQLite</a> plugin.</li>
         </ul>
+        <div class="underline"></div>
+        <h2 class="heading-2">Swift-Kuery-ORM</h2>
+        <p><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-ORM">Swift-Kuery-ORM</a> is an Object Relational Mapping library (ORM) built for Swift. The ORM provides APIs which simplify the persistence of model objects, by allowing you to interact with the database directly from your Swift types (structs and classes).</p>
+        <p>The ORM doesn’t require you to directly specify how the data should be represented in the database. You define your Swift type as a class or a struct, extend it to conform to the `Model` protocol (which requires no additional code) and the ORM automatically generates a table schema for you. You can then save, fetch, update and delete directly by calling functions on your Swift type.</p>
+        <p>The ORM is built on top of <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift-Kuery</a>, which means that it is possible to use Kuery to customize SQL queries made to the database, if the functionality of the ORM is insufficient.  It also means that any database supported by Kuery can be used.</p>
+        <div class="underline"></div>
         <h2 class="heading-2">NoSQL Databases</h2>
         <p class="block-text">NoSQL databases are document based, key-value pairs, graph databases or wide-column stores which do not have standard schema definitions which they need to adhere to unlike SQL databases.</p>
-        <p class="block-text">Kitura has support for the following NoSQL database plugins:</p>
+        <p class="block-text">Kitura has support for the following NoSQL databases:</p>
         <ul>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/Kitura-CouchDB">CouchDB</a></li>
+          <li><a target="_blank" href="http://couchdb.apache.org">CouchDB</a>/Cloudant using the <a target="_blank" href="https://github.com/IBM-Swift/Kitura-CouchDB">Kitura-CouchDB</a> plugin.</li>
           <div class="info">
             <p>This will also work with <a target="_blank" href="https://www.ibm.com/uk-en/cloud/cloudant">IBM Cloudant</a>, as Cloudant uses the same API as CouchDB.</p>
```

Comment:
Now the you link CouchDB/Cloudant, this info box doesn't make as much sense. You could link to cloudant above and say:
"CouchDB and Cloudant use the same API so the Kitura-CouchDB plugin works with either database."

Reply:
👍